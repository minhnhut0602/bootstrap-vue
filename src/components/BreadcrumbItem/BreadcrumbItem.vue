<template>
    <li
        class="breadcrumb-item"
        :class="classes"
    >
        <slot />
    </li>
</template>

<script lang="ts" setup>
import useActive, {activeProps} from '../../composables/useActive';
import useClasses from '../../composables/useClasses';

const props = defineProps({
    ...activeProps,
});

const {classes} = useClasses([
    useActive(props.active).activeClass.value,
]);
</script>
