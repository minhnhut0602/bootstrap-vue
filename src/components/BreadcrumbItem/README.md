# BreadcrumbItem

```html
<BreadcrumbItem active>
    Products
</BreadcrumbItem>
```

## Props

| Name | Type | Description |
| - | - | - |
| active | boolean | Adds the `active` class. |

## Slots

| Name | Description |
| - | - |
| default | Displays content inside the breadcrumb item. |
