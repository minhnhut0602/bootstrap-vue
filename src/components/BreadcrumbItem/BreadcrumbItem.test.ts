import BreadcrumbItem from '.';

describe('template', () => {
    componentRenderTest(BreadcrumbItem);

    componentSlotRenderTest(BreadcrumbItem);

    componentWrapperClassTest(BreadcrumbItem, {active: true}, 'active');
});
