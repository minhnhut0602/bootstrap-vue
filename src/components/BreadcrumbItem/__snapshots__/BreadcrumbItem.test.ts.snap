// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template renders default 1`] = `
"<li class=\\"breadcrumb-item\\">
  <!-- @slot Displays content inside the breadcrumb item -->
</li>"
`;
